### PR TITLE
[MOS-196] Add days of the week to the list of SMS, calls and notes

### PR DESCRIPTION
**Pull request by Maciej-Mudita**
If an event (e.g. receiving a call or SMS) happened in the current week, we display the days of the week instead of the date on the list.

**Your checklist for this pull request**

Make sure that this PR:
- [x] Complies with our guidelines for contributions
- [x] Has unit tests if possible
- [ ] Has documentation updated
- [x] Has changelog entry added

### CODE REVIEW

File: module-utils/time/test/unittest_time.cpp [hunk 1]
```diff
@@ -342,6 +346,55 @@ TEST_CASE("DateTime")
...
+            auto newTimeTimeinfo = currentTimeTimeinfo;
+            SECTION("different day")
+            {
+                auto i = GENERATE(range(7, 32));
+                newTimeTimeinfo.tm_mday -= i;
+                auto newTime = std::mktime(&newTimeTimeinfo);
+                DateTime datetime(timeSettings, newTime, currentTime);
+                REQUIRE(!datetime.isCurrentWeek());
+            }
+            SECTION("different month")
```
Comment (marziel) [inline module-utils/time/test/unittest_time.cpp:371]: Test names like this don't tell much about what's the test supposed to do :( I can see here we have generating months January-December and... what happens next? I mean the test name should tell what's checked without diving into the code under test

Comment (Maciej-Mudita) [reply module-utils/time/test/unittest_time.cpp:371]: If you look at the whole test, can't you see that we change the month in the date to test whether the isCurrentWeek function works correctly?
```
TEST_CASE("DateTime")
...
	SECTION("isCurrentWeek")
	...
		SECTION("not current week")
		...
			SECTION("different month")
			...
```

Comment (marziel) [reply module-utils/time/test/unittest_time.cpp:371]: No :D Anyway, I won't insist - leave it as is if you prefer

Comment (Maciej-Mudita) [reply module-utils/time/test/unittest_time.cpp]: Fixed :)

**Review by SP2FET — APPROVED** [review]

**Review by Lefucjusz — APPROVED** [review]

**Pull request merged**